src/pages/objkt/[...params].tsx

```diff
@@ -101,7 +101,7 @@ const ObjktDetails: NextPage<Props> = ({ objkt }) => {
                 <Button
                   isLink={true}
                 >
-                  see generative token
+                  See collection
```

Comment:
I find this update unclear.
The collection of a Generative Token is already a page (/generative/:id/collection)
This link points to the Generative Token itself, not the collection